torch/_dynamo/bytecode_transformation.py

```diff
@@ -56,6 +56,71 @@ def create_instruction(name, arg=None, argval=_NotProvided, target=None):
         opcode=dis.opmap[name], opname=name, arg=arg, argval=argval, target=target
     )
 
+# Python 3.11 remaps
+def create_jump_absolute(target):
+    if sys.version_info >= (3, 11):
+        return create_instruction("JUMP_FORWARD", target=target)
+    return create_instruction("JUMP_ABSOLUTE", target=target)
```

Comment:
Could be more pythonic and readable, here and everywhere below

```suggestion
def create_jump_absolute(target):
    inst = "JUMP_FORWARD" if sys.version_info >= (3, 11) else "JUMP_ABSOLUTE"
    return create_instruction(inst, target=target)
```